### PR TITLE
Isis 2878

### PR DESCRIPTION
**Pull request by joerg-rade**
The kroviz jar created via `mvn clean install -DskipTests ` is not the same as the one created via `gradle jar`.
Eventually the use of maven-antrun-plugin could be omitted all together (or replaced by maven-dependency-plugin).

The most elegant way to integrate would be a simple addition of 
```
<dependency>
    <groupId>org.apache.isis.incubator.clients</groupId>
     <artifactId>isis-client-kroviz</artifactId>
     <version>${project.version}</version>
</dependency>
```
to demo/domain/pom.xml - once the jar is in the classpath, resources under /public should be available.

### CODE REVIEW

File: examples/demo/domain/pom.xml [hunk 1]
```diff
@@ -27,29 +27,63 @@
...
-				<includes>
-					<!-- we include all .java too, so that we can reference it from the descriptions -->
-					<include>**</include>
-				</includes>
-			</resource>
-		</resources>
-	</build>
+    <build>
+        <plugins>
+            <plugin>
```
Comment (andi-huber) [inline examples/demo/domain/pom.xml:32]: Maybe add a short explanatory comment, for the motivation of adding this plugin. (I guess some copy&past from this PR's description should suffice)

File: examples/demo/pom.xml [hunk 2]
```diff
@@ -238,12 +243,12 @@
 			</activation>
 			<modules>
 				<module>domain</module>
-				<module>javafx</module>
+				<!--module>javafx</module-->
```
Comment (andi-huber) [inline examples/demo/pom.xml:246]: any reason why this was removed?

File: examples/demo/pom.xml [hunk 3]
```diff
@@ -238,12 +243,12 @@
...
 				<module>web</module>
 				<module>wicket/common</module>
 				<module>wicket/jdo</module>
 				<module>wicket/jpa</module>
-				<module>vaadin</module>
+				<!--module>vaadin</module-->
```
Comment (andi-huber) [inline examples/demo/pom.xml:251]: any reason why this was removed?

**Review by andi-huber — COMMENTED** [review]
Basically looks good to me, I added some comments though.

File: examples/demo/pom.xml (hunk 2 above)
Comment (joerg-rade) [reply examples/demo/pom.xml:246]: couldn't build locally - committed by accident

File: examples/demo/pom.xml (hunk 3 above)
Comment (joerg-rade) [reply examples/demo/pom.xml:251]: couldn't build locally - committed by accident

**Pull request merged**